src/rubrix/server/tasks/text_classification/service/service.py

```diff
@@ -337,6 +337,7 @@ def compute_rule_metrics(
             param will be omitted
         label:
             Label used for rule metrics. If not provided, defined in stored rule will be used
+            if rule is found, else related label metrics won't be computed
```

Comment:
Not sure if I understand this, maybe we can rewrite this?
and do we still need this first `if label is None` check in [line 349](https://github.com/recognai/rubrix/pull/907/files#diff-4e658dba045835ec32c8251b1f51f608b56971dfc55d40d368e559f595e76f5eR351-R355), since `self.__labeling__.compute_rule_metrics` can now handle `label=None`?

Reply:
There are 2 cases when label is not provided:

1. Query exists as rule. The label will be obtain from stored label. It's why the line 349 make sense.
2. Query does not exist. No label will be used.